src/switch_rtp.c

```diff
@@ -7827,6 +7826,9 @@ static switch_status_t process_rtcp_report(switch_rtp_t *rtp_session, rtcp_msg_t
 
 			if (report_data.xr_count > 0) {
 				switch_safe_free(report_data.rtcp_data.xr_blocks)
+				if (!report) {
```

Comment:
What are the possibility of xr_count is zero and we received XR report only? I think this should outside of this condition

Reply:
you are right, XR can contain zero reports and still be a valid package